hbase-client/src/main/java/org/apache/hadoop/hbase/quotas/QuotaRetriever.java

```diff
@@ -55,19 +55,29 @@ public class QuotaRetriever implements Closeable, Iterable<QuotaSettings> {
   /**
    * Should QutoaRetriever manage the state of the connection, or leave it be.
    */
-  private boolean isManagedConnection = false;
+  private final boolean isManagedConnection;
 
-  QuotaRetriever() {
+  public QuotaRetriever(final Connection conn) throws IOException {
+    this(conn, (QuotaFilter) null);
```

Comment:
Ah, missed while reviewing the master branch PR. 

Nit: We dont need to type cast the NULL

Reply:
I believe it is required. Without it, Java won't know which version of the two-arg constructor to call -- there are two variants that have `Connection` as the first argument. Indeed, IntelliJ complains when I remove this cast.